src/sbi_cove.adoc

```diff
@@ -770,10 +770,16 @@ enum tsm_state {
     TSM_READY = 2
 };
 
-#define COVE_TSM_CAP_PROMOTE_TO_TVM       0x1
-#define COVE_TSM_CAP_LOCAL_ATTESTATION    0x2
-#define COVE_TSM_CAP_AIA                  0x3
-#define COVE_TSM_CAP_DYN_PAGE_CONVERSION  0x4
+/* Below specifies bit position in the `tsm_capabilities` field */
+#define COVE_TSM_CAP_PROMOTE_TVM         0x0
+#define COVE_TSM_CAP_ATTESTATION_LOCAL   0x1
+#define COVE_TSM_CAP_ATTESTATION_REMOTE  0x2
+#define COVE_TSM_CAP_AIA                 0x3
```

Comment:
In the review today we discovered that a capability is missing. We need to add:
#define COVE_TSM_MRIF 0x4
and chang the memory allocation to 0x5

Reply:
I modified it accordingly.